### PR TITLE
Nav Bar 

### PR DESCRIPTION
**Pull request by jshenny**
Navigation bar and basic main page layout
image
image

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+@media (min-width: 1200px) { ... }
+*/
+
+
+
+
+
+
+
+
```
Comment (JessieG123) [inline src/style.css:182]: Excess spacing

File: src/about/about.html [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+            </div>
+        </div>
+    </body>
+</html>
+<!--- Footer -->
+<footer class="container-fluid">
+	<p>HackTogether 2020 </p>
+	<p>© Covid Overflow 2020</p>
+
+</footer>
```
Comment (JessieG123) [inline src/about/about.html:191]: Should have new line as best practice

**Review by JessieG123 — CHANGES_REQUESTED** [review]

**Review by JessieG123 — COMMENTED** [review]
Otherwise looks good to me!

**Review by ahfhafh — APPROVED** [review]

**Review by JessieG123 — APPROVED** [review]
Can merge

**Pull request closed without merging**